pkg/authorization/authorizer/attributes.go

```diff
@@ -20,6 +20,14 @@ type DefaultAuthorizationAttributes struct {
 	URL               string
 }
 
+func ToDefaultAuthorizationAttributes(in authorizationapi.AuthorizationAttributes) DefaultAuthorizationAttributes {
```

Comment:
> It's hard to reason about why dropping namespace here makes sense - comment or something similar would be good.

Comment added.  I blamed that @smarterclayton guy for saying to simply carry it around on the context :)